docroot/themes/custom/uids_base/templates/field/field.html.twig

```diff
@@ -48,53 +48,29 @@
   label_display == 'visually_hidden' ? 'visually-hidden',
 ] %}
 
-{% if entity_type == 'block_content' %}
```

Comment:
@bspeare Looking back, it looks like you added this section as part of the brand.uiowa.edu updates. I removed it and did a little checking and it seems like maybe it may not be necessary anymore. Can you review this and check to see that I haven't broken whatever you changed this to solve?

Reply:
Yes, it is still necessary until this happens: https://github.com/uiowa/uiowa/issues/1484